src/analyzer/Main.java

```diff
@@ -1,7 +1,17 @@
 package analyzer;
 
+import java.io.IOException;
+import java.util.Arrays;
+
 public class Main {
-    public static void main(String[] args) {
-        System.out.print("Hello world!");
+
+  public static void main(String[] args) throws IOException {
```

Comment:
I think it is better to organize some exception handling here to give a litter descriptive feedback for users of your program. They may not be programmers to deal with exceptions in stdout.